srcpkgs/CLion/update

```diff
@@ -0,0 +1,2 @@
+pattern="CLion-\K[\d.]+(?=\.tar)"
```

Comment:
`CLion` is now pkgname so pattern is not necessary.

Reply:
Thanks for pointing it out.
I tried removing the pattern, but the document the update site returns contains links for all OS, not just Linux.
`xbps-src update-check CLion` then returns something like this:
```
=> Using `/home/aasoft/projects/void_linux_packages/void-packages/hostdir/binpkgs/2asoft/jetbrains-intellij-pro' as local repository.
CLion-2019.1.2 -> CLion-2019.1.3
CLion-2019.1.2 -> CLion-2019.1.3.win
```
This probably doesn't break anything, but if JetBrains ever release non-Linux releases before the Linux releases, the above command would falsely claim that an update is available.

I think it'd be better explicitly have this pattern that is tailored to the Linux releases.

Reply:
There is `ignore=` for that, but you can also leave is as other templates do need pattern anyway.